drivers/hud_pit/app/models/hud_pit/generators/pit/fy2024/base.rb

```diff
@@ -0,0 +1,489 @@
+            next unless member
+
+            member['veteran_status'] == 1 && member['relationship_to_hoh'] == 1
+          end.present?
+
+          options = {
+            client_id: source_client.id,
+            destination_client_id: last_service_history_enrollment.client_id,
+            data_source_id: source_client.data_source_id,
+            report_instance_id: @report.id,
+
+            age: age,
+            dob: source_client.DOB,
+            first_name: source_client.FirstName,
+            last_name: source_client.LastName,
+            household_type: household_type,
+            max_age: household_ages.compact&.max,
+            household_member_count: household_ages.count,
+            hoh_age: hoh_age(hh_id, @generator.filter.on),
+            hoh_veteran: hoh_veteran,
+            head_of_household: enrollment.RelationshipToHoH == 1,
+            relationship_to_hoh: enrollment.RelationshipToHoH,
+            more_than_one_gender: more_than_one_gender,
+            female: source_client.Woman,
+            male: source_client.Man,
+            culturally_specific: source_client.CulturallySpecific,
+            different_identity: source_client.DifferentIdentity,
+            non_binary: source_client.NonBinary,
+            transgender: source_client.Transgender,
+            questioning: source_client.Questioning,
+            gender_none: source_client.GenderNone,
+            pit_race: pit_race,
+            pit_gender: pit_gender,
+            am_ind_ak_native: source_client.AmIndAKNative,
+            asian: source_client.Asian,
+            black_af_american: source_client.BlackAfAmerican,
+            native_hi_other_pacific: source_client.NativeHIPacific,
+            white: source_client.White,
+            mid_east_n_african: source_client.MidEastNAfrican,
+            race_none: source_client.RaceNone,
+            veteran: source_client.VeteranStatus,
+            chronically_homeless: enrollment.chronically_homeless_at_start?(date: @generator.filter.on),
+            chronically_homeless_household: hoh_enrollment&.chronically_homeless_at_start?(date: @generator.filter.on),
+            substance_use: disabilities_latest.detect(&:substance?)&.DisabilityResponse&.present?,
+            substance_use_indefinite_impairing: disabilities_latest.detect { |d| d.indefinite_and_impairs? && d.substance? }&.DisabilityResponse.present?,
+            domestic_violence: dv_record&.DomesticViolenceVictim,
+            domestic_violence_currently_fleeing: dv_record&.CurrentlyFleeing,
+            hiv_aids: disabilities_latest.detect(&:hiv?)&.DisabilityResponse&.present?,
+            mental_illness: disabilities_latest.detect(&:mental?)&.DisabilityResponse&.present?,
+            mental_illness_indefinite_impairing: disabilities_latest.detect { |d| d.indefinite_and_impairs? && d.mental? }&.DisabilityResponse.present?,
+            project_type: last_service_history_enrollment.computed_project_type,
```

Comment:
This can now just be:
```suggestion
            project_type: last_service_history_enrollment.project_type,
```
though it should not affect the output any more